### PR TITLE
Rename 'parcel' to 'lot'

### PR DESCRIPTION
**Pull request by klochowicz**
Lot size seems to be the correct domain term.

### CODE REVIEW

File: daemon/src/projection.rs [hunk 1]
```diff
@@ -979,26 +979,26 @@ pub struct CfdOrder {
...
-    pub margin_per_parcel: Amount,
+    pub margin_per_lot: Amount,
 
-    /// Initial funding fee per parcel from the perspective of the role
+    /// Initial funding fee per lot from the perspective of the role
     ///
     /// Since this is a calculated value that we need in the UI this value is based on the
     /// perspective the role (i.e. taker/maker)
     #[serde(with = "::bdk::bitcoin::util::amount::serde::as_btc")]
-    pub initial_funding_fee_per_parcel: SignedAmount,
```
Comment (thomaseizinger) [inline daemon/src/projection.rs:996]: Are any of these fields used by `autotaker` or other HTTP clients? If yes, then we might want to consider duplicating them with a new field name first and update all clients before we remove them.

Comment (klochowicz) [reply daemon/src/projection.rs:996]: great idea. I will do that before I merge this PR - I posted this first to get the buy-in from everyone first.

Comment (klochowicz) [reply daemon/src/projection.rs:996]: I have checked the autotaker and automation, none of them care about parcel so far :)

**Comment by thomaseizinger** [comment]
concept ACK

**Review by da-kami — APPROVED** [review]

**Pull request merged**